ftw/noticeboard/browser/templates/noticeboard.pt

```diff
@@ -0,0 +1,22 @@
+<html xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      metal:use-macro="context/main_template/macros/master"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      i18n:domain="ftw.noticeboard">
+
+    <div metal:fill-slot="content-core">
+        <tal:categories repeat="category view/get_categories_and_notices">
```

Comment:
I think a class would help here to apply styles later on.
It is against YAGNI but can be very helpful. Maybe even wrap the whole stuff into its own container.

Reply:
That's a good point. I'll add it once I apply some styles. It depends a little bit on the JS integration.